demos/aquarium/src/App.jsx

```diff
@@ -8,7 +8,7 @@ import turtleModel from './model_52a_-_kemps_ridley_sea_turtle_no_id-transformed
 
 export default function App({ spheres }) {
   return (
-    <Canvas shadows camera={{ position: [30, 0, -3], fov: 35, near: 1, far: 50 }}>
+    <Canvas shadows camera={{ position: [30, 0, -3], fov: 35, near: 1, far: 50 }} gl={{ stencil:true }}>
```

Comment:
is this `gl={{ stencil:true }}` really necessary in this case? I tried without, seems to have no effect :/

Reply:
what's default value?

Reply:
without stencil: true
![Screenshot from 2024-07-31 12-28-02](https://github.com/user-attachments/assets/354b07aa-77fd-4422-a341-3129b3961705)

with stencil:true
![Screenshot from 2024-07-31 12-26-49](https://github.com/user-attachments/assets/281ab781-a805-405c-b17c-853a061196a7)